db/seeds/playbook.seeds.rb

```diff
@@ -95,6 +95,41 @@
 strong enough to fight, but I studied, trained, and
 now I’m ready to cleanse the world of their taint.
 I’ll kill them all. That’s all I have left.'
+<<<<<<< HEAD

```

Comment:
This is from a Merge Conflict. It needs to be removed/resolved, before this can be deployed.